docs/src/examples/lorenz_example.md

```diff
@@ -1,26 +1,289 @@
+savefig(figpath)
+figpath = joinpath(figure_save_directory, "prior" * ".png")
+savefig(figpath)
+```
+
+```@raw html
+<img src="../../assets/Lorenz-prior.png" width="600">
+```
+
+Observational Noise: The observational noise can be generated using the L96 system or prescribed, as specified by `var_prescribe`. 
+
+`var_prescribe==false`
+The observational noise is constructed by generating independent instantiations of the L96 statistics of interest at the true parameters for different initial conditions.
+The empirical covariance matrix is constructed.
+
+`var_prescribe==true`
+The observational noise is prescribed as a Gaussian distribution with prescribed mean and variance.
+
+
+Machine learning emulation settings: We have 9 cases that the user can toggle or customize
+```julia
+cases = [
+        "GP", # diagonalize, train scalar GP, assume diag inputs
+        "RF-scalar-diagin", # diagonalize, train scalar RF, assume diag inputs (most comparable to GP)
+        "RF-scalar", # diagonalize, train scalar RF, do not asume diag inputs
+        "RF-vector-svd-diag",
+        "RF-vector-svd-nondiag",
+        "RF-vector-nosvd-diag",
+        "RF-vector-nosvd-nondiag",
+        "RF-vector-svd-nonsep",
+]
+```
+The first is for GP with `GaussianProcesses.jl` interface. The next two are for the scalar RF interface, which most closely follows exactly replacing a GP. The rest are examples of vector RF with different types of data processing, (svd = same processing as scalar RF, nosvd = unprocessed) and different RF kernel structures in the output space of increasing complexity/flexibility (diag = Separable diagonal, nondiag = Separable nondiagonal, nonsep = nonseparable nondiagonal).
+
+The example then loads the relevant training data that was constructed in the `calibrate.jl` call. 
+```julia 
+# loading relevant data
+homedir = pwd()
+println(homedir)
+figure_save_directory = joinpath(homedir, "output/")
+data_save_directory = joinpath(homedir, "output/")
+data_save_file = joinpath(data_save_directory, "calibrate_results.jld2")
+ekiobj = load(data_save_file)["eki"]
+priors = load(data_save_file)["priors"]
+truth_sample_mean = load(data_save_file)["truth_sample_mean"]
+truth_sample = load(data_save_file)["truth_sample"]
+truth_params_constrained = load(data_save_file)["truth_input_constrained"] #true parameters in constrained space
+truth_params = transform_constrained_to_unconstrained(priors, truth_params_constrained)
+Γy = ekiobj.obs_noise_cov
+```
+
+We then set up the structure of the emulator. An example for GP (`GP`)
+```julia
+gppackage = Emulators.GPJL()
+pred_type = Emulators.YType()
+mlt = GaussianProcess(
+    gppackage;
+    kernel = nothing, # use default squared exponential kernel
+    prediction_type = pred_type,
+    noise_learn = false,
+)
+```
+
+An example for scalar RF (`RF-scalar`)
+```julia
+n_features = 100
+kernel_structure =
+case == "RF-scalar-diagin" ? SeparableKernel(DiagonalFactor(nugget), OneDimFactor()) :
```

Comment:
Could you remove the switch with `RF-scalar` and `RF-scalar-diagin` and so the snippet just reads 
```julia
kernel_structure = SeparableKernel(LowRankFactor(2, nugget), OneDimFactor())
```